src/lib/structures/Command.ts

```diff
@@ -106,44 +93,40 @@ export abstract class Command<T = Args> extends AliasPiece {
 		return new Args(message, this as any, args, context) as any;
 	}
 
-	/**
-	 * Get all the main categories of commands.
-	 */
-	public get categories(): (string | null)[] {
-		return Array.from(new Set([...this.container.stores.get('commands').values()].map(({ category }) => category)));
-	}
-
 	/**
 	 * The main category for the command, if any.
-	 * This is resolved from {@link Command.fullCategory}, which is automatically
-	 * resolved in the constructor. If you need different logic for category
-	 * then please first look into overwriting {@link Command.fullCategory} before
-	 * looking to overwrite this getter.
+	 *
+	 * This getter retrieves the first value of {@link Command.fullCategory} if has at least one item, otherwise it
+	 * returns `null`.
```

Comment:
```suggestion
	 * This getter retrieves the first value of {@link Command.fullCategory}, if it has at least one item, otherwise it
	 * returns `null`.
```